app/packages/operators/src/built-in-operators.ts

```diff
@@ -1211,6 +1212,56 @@ export class SetPanelTitle extends Operator {
   }
 }
 
+type SetPlayheadStateHooks = {
+  setPlayheadState: (state: fop.PlayheadState, timeline_name?: string) => void;
+};
+type SetPlayheadStateParams = { state: any; timeline_name?: string };
+export class SetPlayheadState extends Operator {
```

Comment:
nit: whitespace

and in line 1218, can state not be typed `fop.PlayheadState`?

also we're taking in `timeline_name` but not doing anything with it...